### PR TITLE
Task#18 - Added a header & changed a package for material icons

### LINKED ISSUE #18: As a user I want to navigate into app via header
### As a user I want to navigate into app via header

- [x]  Navigate to:  profile, search, chat, settings, login pages.
- [x] Parse data from request and display a userIcon
- [x] Open a dropdown menu by clicking on userIcon and close It by clicking on userIcon or outside of the menu.
- [ ] Add red circle for chatIcon if a user has notification ( new message ) 
- [x]  Create design for mobile

### PR DESCRIPTION
**Pull request by grok-rs**

### CODE REVIEW

**Comment by oleksandrstarov** [comment]
is this related to #18 @dromix ?

**Review by Pr1sm0 — APPROVED** [review]

File: frontend/src/components/shared/header/Header.tsx [hunk 1]
```diff
@@ -1 +1,42 @@
-import React from 'react';
\ No newline at end of file
+import MailOutlineIcon from '@material-ui/icons/MailOutline';
+import SearchIcon from '@material-ui/icons/Search';
+import React, { useState } from 'react';
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/header/Header.tsx:3]: please move the React import to the top so it is aligned with other files

File: frontend/src/components/shared/header/Header.tsx [hunk 2]
```diff
@@ -1 +1,42 @@
...
+import { Link } from 'react-router-dom';
+import './Header.scss';
+import logo from './logo.png';
+
+const Header: React.FC = () => {
+  const [showMenu, setMenu] = useState('');
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/header/Header.tsx:9]: please follow naming for state [showMenu, setShowMenu]
From my point of view it is better to name this var [menuClass, setMenuClass] for this solution

File: frontend/src/components/shared/header/Header.tsx [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+  const iconName = 'icon_0';
+
+  const toggleMenu = () => {
+    setMenu(showMenu ? '' : 'dropdown-menu-active');
```
Comment (oleksandrstarov) [inline frontend/src/components/shared/header/Header.tsx:14]: Please wrap in `Boolean(showMenu)` for better readability

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**